test/Feature/CBuffer/lit.local.cfg

```diff
@@ -0,0 +1,6 @@
+if 'Clang' in config.available_features:
+    config.unsupported = True
+# Disable these tests under metal until we've resolved issues with cbuffer
+# binding access.
```

Comment:
Can you file an issue so that we can track this?

Reply:
Filed #55 and updated the comment to reference it.